Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/TSXL-Jiva-controller-network-delay/controller-network-delay

```diff
@@ -0,0 +1,185 @@
+################
+
+: << EOF
+  --------------------------------------------------------------------------------------------------------------------     
+ | specAttribute     | kind   |         default value               | test specifc value                              |
+  ------------------------------------------------------------------------------------------------------------------- |
+ | appLabel          | env    | app=busybox-sts                     | app=busybox-controller-delay-jiva               |
+ | deploy type       | env    | statefulset                         | deployment                                      |
+ | appNamespace      | env    | app-busybox-ns                      | controller-delay-jiva                           |
+ | pvcName           | env    | openebs-busybox                     | openebs-bb-controller-delay                     | 
+ | storage clas      | env    | openebs-cstor-sparse                | openebs-jiva-default                            |
+ | Litmus Job name   | name   | generateName: litmus-busybox-deploy | generateName: #busybox-mem-check-deprovision-jiva|
+ | Litmus job label  | label  | app: busybox-litmus                 | mem-busybox-jiva                                |
+  ---------------------------------------------------------------------------------------------------------------------
+EOF
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-provision-controller-delay-jiva metadata="")
+echo $test_name
+
+cd litmus
+echo "Running the litmus test for Busybox Deployment.."
+
+cp apps/busybox/deployers/run_litmus_test.yml busybox_target_delay.yml
+
+
+
+sed -i -e 's/generateName: litmus-busybox-deploy/generateName: busybox-provision-controller-delay-jiva/g' \
+-e 's/app: busybox-litmus/app: busybox-controller-delay-jiva/g' \  
+-e 's/app=busybox-sts/app=controller-delay-jiva/g' \   
+-e 's/value: statefulset/value: deployment/g' \
+-e 's/value: openebs-cstor-sparse/value: openebs-jiva-default/g' \
+-e 's/value: openebs-busybox/value: openebs-bb-controller-delay/g' \
+-e 's/value: app-busybox-ns/value: controller-delay-jiva/g' busybox_target_delay.yml
+cat busybox_target_delay.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:busybox-controller-delay-jiva' job=busybox_target_delay.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:controller-network-delay-jiva $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id TSXL 4-chaos "Induce delay in jiva controller pod network and verify if the application is not impacted" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+echo "*******Performing network delay on controller**********"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=openebs-controller-network-delay-jiva metadata="")  ####################
```

Comment:
@harshita-sharma-mayadata `testcase` is the test name that we defined in the litmus test_vars.yml. can you update the test case name for jiva with the metadata?